src/python/gem5/components/boards/lupv_board.py

```diff
@@ -94,14 +95,20 @@ def __init__(
 
         self._excep_code = { 'INT_SOFT_SUPER': 1, 'INT_TIMER_SUPER': 5,
                              'INT_TIMER_MACHINE': 7, 'INT_EXT_SUPER': 9,
-                             'INT_EXT_MACHINE': 10 }
+                             'INT_EXT_MACHINE': 11 }
```

Comment:
Not a big big deal but the current formatting of this dict is quite confusing (I struggled with it for a couple minutes when I was working on this code last week). The interrupt number related to the same event type should ideally be on the same line, for symmetry:

```python
        self._excep_code = { 'INT_SOFT_SUPER'  : 1,
                             'INT_TIMER_SUPER' : 5, 'INT_TIMER_MACHINE' : 7,
                             'INT_EXT_SUPER'   : 9, 'INT_EXT_MACHINE'   : 11 }
```

Makes the array much easier to read IMO :)